manifests/init.pp

```diff
@@ -119,6 +125,8 @@
   Boolean $quiet                       = $telegraf::params::quiet,
   Hash    $inputs                      = $telegraf::params::inputs,
   Hash    $outputs                     = $telegraf::params::outputs,
+  Hash    $processors                  = $telegraf::params::processors,
```

Comment:
since those defaults are identical across every operating system, we don't need to define it within hiera:
```suggestion
  Hash    $processors                  = {},
```

Are you able to enforce the datatypes within the hash?

Reply:
The current ```params.pp``` does not contain the parameters ```aggregators``` or ```processors```.